inc/item_softwareversion.class.php

```diff
@@ -1558,8 +1558,15 @@ static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtem
 
 
    protected static function getListForItemParams(CommonDBTM $item, $noent = false) {
-      $params = parent::getListForItemParams($item, $noent);
```

Comment:
Not sure of this. The parent method was populating more than just a `WHERE` clause. This method `getListForItemParams` is also used by `CommonDBRelation::getListForItem()`. It does not seems to be used directly for computers in GLPI, but it may be used in plugins.

See alternative solution on next comment.